### ISSUE TITLE
noetic: obstacle avoidance with the RGB-D camera seems to be disabled 

### ISSUE THREAD

**Issue by ah96** [pal-robotics/tiago_tutorials#42]
Launching tiago_navigation in Gazebo from the tiago_2dnav_gazebo package on ROS Noetic (Localization and Path Planning tutorial: http://wiki.ros.org/Robots/TIAGo/Tutorials/Navigation/Localization), TIAGo is constantly hitting tables, even in the simple_office world. 
On ROS Melodic this does not happen.

Is obstacle avoidance with the RGB-D camera disabled by default in the Noetic version?

If yes, how can it be enabled?

**Comment by saikishor** [comment]
Hello @ah96,

Thank you for showing interest in our TIAGo simulation. We will try to reproduce this issue in the upcoming days and get back to you.

Thank you,

Best Regards,
PAL Robotics Team.

**Comment by saikishor** [comment]
Hello @ah96,

I've consulted with the navigation team, we don't have the table avoidance available in the public simulation. We have tried to reproduce the same at our end with Noetic and Melodic and it does collide in both cases. We have an advanced navigation package available for our customers that does use the RGB-D camera for table avoidance. I hope this answers your query.

Best Regards,
PAL Robotics Team.
